src/repository/Repository.ts

```diff
@@ -345,5 +345,20 @@ export class Repository<Entity extends ObjectLiteral> {
     async clear(): Promise<void> {
         return this.manager.clear(this.metadata.target);
     }
-
+    
+    /**
+     * Reloads the supplied entity into the latest state of the record from the database
+     */
+    reload(entity: Entity): Promise<void> {
+        return new Promise(async (resolve, reject) => {
```

Comment:
Promise can be rewritten to async reload function.

Reply:
Fair point, I'll change it